migrator/migrations/2_add_index_on_user_emails.up.sql

```diff
@@ -0,0 +1 @@
+CREATE UNIQUE INDEX users_email_idx ON public.users (email);
```

Comment:
Why do we need an index for emails?

Reply:
Read more about difference of constraint and index, and in that case it will be better constraint. Fix it